common/version.go

```diff
@@ -3,7 +3,7 @@ package common
 import "context"
 
 var (
-	version = "0.4.8"
+	version = "0.4.9"
```

Comment:
leave this out - it's usually done separately. For development you can use this: https://www.terraform.io/cli/config/config-file#development-overrides-for-provider-developers